src/main/java/edu/kit/ipd/crowdcontrol/objectservice/config/Deployment.java

```diff
@@ -5,7 +5,11 @@
  */
 public class Deployment {
     /**
-     * Deployment origin of the control ui.
+     * Port to listen on
+     */
+    public Integer port;
```

Comment:
mhmm if I remember correctly you can add the default value after this field, if no config value is set for it, the default is used. (and isnt a port a ushort?)